### DISCUSSION TITLE
NBreaks variables

### DISCUSSION THREAD

**Discussion by JerryOhlin** [Ergo-Tools/ActiPASS#62] [Help and Q&A (processing, quality checking or statistics)] [answered]
We are investigating physical activity in 130 active workers (construction, cleaning staff, commercial kitchen, assistant nurses) with around 6 days measurements (24 hour wear protocol, at least 20 h/day to be valid). 
We want to report the number of breaks from activities, i.e. how many times they can sit down and rest. 
From what i gather in the actiPASS variable definition the "Sitlie_Nbreaks" is the only one that is recommended to be used without consulting the developers. Interestingly, in our data, the variables "Upright_NBreaks" is almost identical to "SitLie_Nbreaks" (see table below)

SitLie_NBreaks      Upright_NBreaks          
 Min.   :  2.00         Min.   :  2.0   
 1st Qu.: 50.00      1st Qu.: 50.0   
 Median : 67.00     Median : 67.0   
 Mean   : 77.62      Mean   : 77.5   
 3rd Qu.: 87.25      3rd Qu.: 87.0                       
 Max.   :354.00       Max.   :355.0   

I was wondering what the difference between these two variables are and if they are equally reliable?

**Comment by pasan-h** [comment, marked as answer]
The NBreaks variable for for a given activity, behaviour or an intensity-class is calculated by counting number of times that activity, behaviour or an intensity-class transits to another. 

For certain activity/behaviour types (such as walking, standing, moving, stair-walking etc.), when we increase time-resolution of the underlying detection algorithm (when the epoch-length becomes smaller), we can expect more and more such transitions (NBreaks) to be detected. This phenomena is also closely related to the seemingly high MVPA minutes per day from ActiPASS/Acti4 compared to a traditional count based algorithm with low time resolution (longer epoch length).

This means for certain activity/behaviour types, interpretation of NBreaks should be done carefully. One can even argue NBreaks calculated with such low epoch-length doesn't make much sense for behaviours such as standing or walking. This is because, for an example, during a typical **"perceived bout of walking"**, algorithm output can have intermittent gaps of "standing" or "moving". And all such gaps will be counted in the "Walk_NBreak" variable. If one really needs to understand the discontinuities of activities like walking, perhaps it makes more sense to  consider a **bout-variable** such as "Walk_1min_freq_H".

However, NBreaks calculated for combined posture **SitLie** or **Upright** (which is defined as all activities/behaviours which are **not "sit" or "Lie"**) should be accurate. Since there is not much ambiguity concerning these two postures (both algorithm perspective or how we perceive them), the detection is robust. Since the variable "Upright" is defined as any activity/behaviour which is not "sit" or "Lie", variables "SitLie_NBreaks" and "Upright_NBreaks" should more or less be equal.  Any differences between the these two variables should be related to what activity/behaviour **was present during midnight**. 

It is also important to keep in mind that behaviour **"Other" and "Cycling"** which is categorized as **Upright** might contain some episodes of a **sitting posture**.

Hope this answer is helpful 😉.

**Reply by JerryOhlin** [reply]
Thank you Pasan, this was very informative!
